.github/workflows/dart_ddc.yaml

```diff
@@ -63,6 +64,8 @@ jobs:
       - name: Setup Firefox
         if: ${{ matrix.browser == 'firefox' }}
         uses: ./.github/composite_actions/setup_firefox
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
question: Does the token used here require some specific permission? Seems like consumer jobs set to `read-all` so I was just curious if there was some specific permission needed for this API? no change needed in PR

Reply:
No special permissions required. It's just to let GitHub know who we are when making a request so we don't get pooled into the IP-based rate limiting bucket.